### PR TITLE
End Registration when the Event Starts

### PR DESCRIPTION
**Pull request by martypowell**
Originally registration would end 30 minutes prior to the event, but we want it to end either when the event starts or spots are full.

### CODE REVIEW

File: js/apps/events-page/dataServices/eventsService.js [hunk 1]
```diff
@@ -82,7 +82,7 @@
 							// Since moment().subtract() mutates the date rather than returning a new date,
 							// we need to calculate the date fresh every time.
 							response.data.registrationStarts = moment(response.data.EventStart).subtract(7, 'days');
-							response.data.registrationEnds = moment(response.data.EventStart).subtract(30, 'minutes');
+							response.data.registrationEnds = moment(response.data.EventStart);
```
Comment (ascire) [inline js/apps/events-page/dataServices/eventsService.js]: Can the time offsets for both `registrationStarts` and `registrationEnds` be fed from a configuration file?

Comment (martypowell) [reply js/apps/events-page/dataServices/eventsService.js]: yes, and it's desired.

**Comment by martypowell** [comment]
@jsmith176 @ascire ready for another look

File: dist/js/apps/events-page/app.js [hunk 2]
```diff
@@ -158,7 +158,7 @@ bcpl.boostrapCollapseHelper = function ($) {
 	var app = angular.module('events', []);
 
 	var constants = {
-		baseUrl: 'https://services.bcpl.info',
+		baseUrl: 'https://testservices.bcpl.info',
```
Comment (ascire) [inline dist/js/apps/events-page/app.js]: Should this have changed?

Comment (martypowell) [reply dist/js/apps/events-page/app.js]: no, good catch, i thought i removed that.

Comment (martypowell) [reply dist/js/apps/events-page/app.js]: @ascire fixed

**Comment by ascire** [comment]
Go forth and merge.

**Pull request merged**